Sources/UIViewExtensions.swift

```diff
@@ -307,6 +309,28 @@ extension UIView {
         self.layer.transform = transform
     }
 
+    
+    /// EZSwiftExtensions
+    public func setTranslation(_ duration:TimeInterval?, _ x: CGFloat?, y: CGFloat?, completion: ((Bool) -> Void)? = nil) {
+        self.animate(duration: duration ?? UIViewTransitionDuration, animations: {
+            self.transform = .init(translationX: x ?? 0, y: y ?? 0)
```

Comment:
I do not understand what these methods actually do. Can you put a comment on the top.